tests/capice/utilities/test_predictor.py

```diff
@@ -24,8 +24,8 @@ def setUpClass(cls):
     def test_predict(self):
         observed = self.predictor.predict(self.dataset)
         expected = pd.Series(
-            [0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5021135210990906]).astype(
-            np.float32).rename('score')
+            [0.20261085, 0.4030959, 0.5546794, 0.71313614]
```

Comment:
are you testing for expected floating point values here? in that case you should never test for equality but always use a delta to compare floats (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual).